greenboot-rs.spec

```
@@ -12,10 +12,16 @@ Summary:	Generic Health Check Framework for systemd
 License:	BSD-3-Clause AND ISC AND MIT AND Unicode-DFS-2016 AND (Apache-2.0 OR BSL-1.0) AND (Apache-2.0 OR MIT) AND (Apache-2.0 WITH LLVM-exception OR Apache-2.0 OR MIT) AND (Unlicense OR MIT)
 URL:		https://github.com/fedora-iot/greenboot-rs
 Source0:	%{url}/releases/download/%{version}/%{name}-%{version}.tar.gz
+Source1:	%{name}-%{version}-vendor-patched.tar.xz
 
 ExcludeArch:	%{ix86}
 
+%if 0%{?centos} && !0%{?eln}
```

Is this correct? shouldn't it be:
```suggestion
%if 0%{?eln} || 0%{?centos} || 0%{?rhel}
```
(same for other occurrences below)
Shouldn't we also enable `ELN` builds in packit?

---

Done.